recipes/stopt/meta.yaml

```diff
@@ -0,0 +1,67 @@
+{% set name = "stopt" %}
+{% set version = "5.12" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  git_url: https://gitlab.com/stochastic-control/StOpt
+  git_rev: "v{{ version }}" # (Defaults to "HEAD")
+  # git_depth: 1 # (Defaults to -1/not shallow)
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - {{ stdlib('c') }}
+    - cmake >=3.15
+    - make
+    - pkgconfig  # [unix]
+  host:
+    - boost  # TODO: revise to libboost-devel in the future
+    - libboost-mpi
+    - mpich  # [unix]
+    - msmpi  # [win]
+    - _openmp_mutex  # [linux]
+    - llvm-openmp  # [osx or win]
+    - eigen
+    - coin-or-utils
+    - coin-or-clp
+    - coin-or-osi
+    - bzip2
+    - zlib
+  run:
+    - libboost
+    - libboost-mpi
+    - mpich  # [unix]
+    - msmpi  # [win]
+    - _openmp_mutex  # [linux]
+    - llvm-openmp  # [osx or win]
+    - eigen
+    - bzip2
+    - zlib
+
+test:
+  requires:
+    - cmake >=3.15
+  commands:
+    - ctest -R MyTestForGlobal  # [not win]
```

Comment:
Please check for the correct installation of the expected files, e.g. libraries.

Reply:
Will do; I'll test the availability of binaries in the export folder

Reply:
Updated